lib/controls/ValueSelector.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class ValueSelector extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {value, options, className, handleOnChange} = this.props;
+    const children = this.props.children;
+
+    if (children) {
+      return (
+        React.cloneElement(children,
```

Comment:
but what if there is no custom control? i.e.
```js
controls = {
  valueEditor: null
}

<QueryBuilder controls={controls} />
```

There won't be a control to render the fallback, right? maybe a short snippet would help me understand better?